### ISSUE TITLE
Butterflyfish interaction data

### ISSUE THREAD

**Issue by salkeith** [globalbioticinteractions/globalbioticinteractions#964]
Hello. I have a dataset with congeneric butterflyfish encounters (>5000) and whether they are passive or aggressive, and butterflyfish feeding data (on scleractinian corals). They have been used in 2 papers so far https://www.nature.com/articles/s41558-018-0314-7 ; https://royalsocietypublishing.org/doi/10.1098/rspb.2022.2158. I have just come across GloBI and thought it might be possible to share the data on there but don't have enough time to figure out how to share (I had quick look Zenodo!). Can you help? Thanks, Sally

**Comment by jhpoelen** [comment]
@salkeith hey Sally - thanks for reaching out. Can you point to the (raw) data that contain the butterfly fish encounters and feeding records?

**Comment by salkeith** [comment]
Ah, I'd forgotten that was there! All the encounter/aggression data is in that repository, although I've also attached here a csv with all aggression data up to July 2017 (we have a bit more now but this is the bulk of it). The feeding data is only on my computer at the moment and attached here as a bunch of CSVs. I have it as a single Rdata file but Github didnt like that.
 
[FeedGenus_BaliNov2015.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481345/FeedGenus_BaliNov2015.csv)
[FeedGenus_ChristmasJunJul2017_JP.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481346/FeedGenus_ChristmasJunJul2017_JP.csv)
[FeedGenus_ChristmasSept2015_JP.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481347/FeedGenus_ChristmasSept2015_JP.csv)
[FeedGenus_ChristmasSept2015.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481348/FeedGenus_ChristmasSept2015.csv)
[FeedGenus_PhilippinesApr2016.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481349/FeedGenus_PhilippinesApr2016.csv)
[FeedSpecies_AcehFeb2017.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481350/FeedSpecies_AcehFeb2017.csv)
[FeedSpecies_BaliJuly2016.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481351/FeedSpecies_BaliJuly2016.csv)
[FeedSpecies_IriomoteJune2016.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481352/FeedSpecies_IriomoteJune2016.csv)
[FeedSpecies_IriomoteJune2017.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481354/FeedSpecies_IriomoteJune2017.csv)
[FeedSpecies_KimbeOct2016.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481355/FeedSpecies_KimbeOct2016.csv)
[FeedSpecies_PhilippinesOct2016.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481356/FeedSpecies_PhilippinesOct2016.csv)
[AggressionDataJuly2017MASTER.csv](https://github.com/globalbioticinteractions/globalbioticinteractions/files/14481357/AggressionDataJuly2017MASTER.csv)

**Comment by jhpoelen** [comment]
@salkeith Hey Sally - thanks for sharing your raw datasets. Would you be open to having GloBI index these data? If so, what would be a good place for your to publish your data is a citable way? I'd prefer publishing the data in a separate github repository that you own. Then, once the repository is looking ok, we can enable it for GloBI indexing. Then, you can choose the publish a "release" or a version of the data to Zenodo. 

Curious to hear your thoughts,
-jorrit

**Comment by salkeith** [comment]
Yes I would :) It is just a case of when I get around to formatting it appropriately. How would you need the csv files set out? Then I can collate those and pop them on a GitHub respository as you suggest. Thanks

**Comment by jhpoelen** [comment]
> Yes I would :)

Great!

>  It is just a case of when I get around to formatting it appropriately. How would you need the csv files set out? Then I can collate those and pop them on a GitHub respository as you suggest.

If you create a github repo with csv files as-is including a README to document their origin, I can try and add schema definitions to register the individual csv files  with GloBI. Then, you can choose to make changes when needed.

I think the important thing is to get the data out as-is, so that others (like me) can chime in and help make the data more accessible. Much easier than going back in time, putting on scuba gear and getting new observations ; )

**Comment by salkeith** [comment]
Okey dokey. Will do as soon as I get chance - probably next week as I have lots of teaching this week

**Comment by salkeith** [comment]
Just getting the final go ahead from others who contributed to the dataset. I don't anticipate any problems...

**Comment by salkeith** [comment]
Apologies I haven’t got any further with this, I haven’t forgotten but have just had a knee op and recovering - will get to it ASAP.
